cli/module_build.go

```diff
@@ -424,7 +424,13 @@ func (c *viamClient) listModuleBuildJobs(moduleIDFilter string, count *int32, bu
 // waitForBuildToFinish calls listModuleBuildJobs every moduleBuildPollingInterval
 // Will wait until the status of the specified job is DONE or FAILED
 // if platform is empty, it waits for all jobs associated with the ID.
-func (c *viamClient) waitForBuildToFinish(buildID, platform string) (map[string]jobStatus, error) {
+// If pm is not nil, it will show progress spinners for each build step.
+func (c *viamClient) waitForBuildToFinish(
+	buildID string,
+	platform string,
+	pm *ProgressManager,
+	displayBuildID string,
```

Comment:
(q) it looks like at every callsite, we just pass the same value for both `buildID` and `displayBuildID`. Why do we need to take both arguments at all?